plugins/@grouparoo/freshdesk/src/lib/client.ts

```diff
@@ -0,0 +1,108 @@
+import axios, { AxiosInstance } from "axios";
+
+type ContactSearchParams = "email" | "phone" | "mobile" | "unique_external_id";
+
+export class FreshdeskClient {
+  readonly axios: AxiosInstance;
+
+  constructor(readonly domainName: string, apiKey: string) {
+    this.axios = axios.create({
+      baseURL: domainName,
+      auth: { username: apiKey, password: "X" },
```

Comment:
This `X` is odd. That the spec?

Reply:
Yeah! It's really odd, but that's how they documented it and how it works!